apps/server/src/services/institutionSyncer.test.ts

```diff
@@ -0,0 +1,135 @@
+import { http, HttpResponse } from 'msw'
+import { clearIntervalAsync } from 'set-interval-async'
+import { server } from '../test/testServer'
+import * as syncerModule from './institutionSyncer'
+import {
+  loadCachedInstitutions,
+  setInstitutionSyncSchedule
+} from './institutionSyncer'
+
+import { writeFileSync } from 'fs'
+import config from '../config'
+import * as logger from '../infra/logger'
+import { elasticSearchInstitutionData } from '../test/testData/institution'
+import * as auth0Module from './auth0Service'
+import * as esModule from './ElasticSearchClient'
+import { getInstitutionFilePath } from './ElasticSearchClient'
+import * as redis from './storageClient/redis'
+
+jest.mock('fs')
+
+describe('setInstitutionSyncSchedule', () => {
+  beforeEach(() => {
+    jest.useFakeTimers()
+  })
+
+  afterEach(() => {
+    jest.useRealTimers()
+    jest.restoreAllMocks()
+  })
+
+  it.only('should call sync over the given interval', async () => {
+    const loadCachedSpy = jest
```

Comment:
Generally we want to mock things at as low of a level as possible. In this case I'd use MSW to check that the request is made